implement-cowsay/cow.py

```diff
@@ -0,0 +1,29 @@
+import cowsay
+import argparse
+
+parser = argparse.ArgumentParser(
+    prog="cowsay",
+    description="Make animals say things",
+)
+
+valid_animal = cowsay.char_names
+
+parser.add_argument(
+    "--animal",
+    choices=valid_animal,
+    default="cow",
+    help=" The animal to be saying things."
+)
+parser.add_argument(
+    "message", 
+    nargs="+", 
+    help="The message to say"
+)
+
+args = parser.parse_args()
+
+# to compose full msg
+message = " ".join(args.message)
+
+# Call selected cowsay func
+getattr(cowsay, args.animal)(message)
```

Comment:
You can check out an alternative `get_output_string` method here to avoid getattr (not the best choice to use if there is alternative - it make code more readable): https://pypi.org/project/cowsay/